test/podFailover/metrics-svc.yaml

```diff
@@ -0,0 +1,34 @@
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  name: pod-failover-metrics
+  labels:
+    app: pod-failover-metrics
+spec:
+  nodeSelector:
+    kubernetes.io/os: linux
+  containers:
+    - image: metrics-service-image
+      name: controller
+      imagePullPolicy: IfNotPresent
+  restartPolicy: OnFailure
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: pod-failover-metrics-service
+  namespace: default
+  labels:
+    app: pod-failover-metrics
+spec:
+  selector:
+    app: pod-failover-metrics
+  ports:
+    - name: metrics
+      port: 9090
+      targetPort: 9090
+    - name: podDowntime
```

Comment:
I get the following error when trying to apply this YAML file:
```console
$ kubectl apply -f test/podFailover/metrics-svc.yaml
pod/pod-failover-metrics created
The Service "pod-failover-metrics-service" is invalid: spec.ports[1].name: Invalid value: "podDowntime": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
Ah I see. I'll quickly fix it. The entire port name should be in small letters. You can change that and it should work.